sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -378,6 +380,47 @@ private[hive] class HiveClientImpl(
 
       val properties = Option(h.getParameters).map(_.asScala.toMap).orNull
 
+      val excludedTableProperties = Set(
+        // The following are hive-generated statistics fields. Currently, only total_size and
+        // row_count are used to populate the dedicated field `stats`.
+        // TODO: stats should include all the other three fields.
+        StatsSetupConst.COLUMN_STATS_ACCURATE,
+        StatsSetupConst.NUM_FILES,
+        StatsSetupConst.NUM_PARTITIONS,
+        StatsSetupConst.ROW_COUNT,
+        StatsSetupConst.RAW_DATA_SIZE,
```

Comment:
`CatalogTable` have a field `unsupportedFeatures`, can we extend it to hide this kind of hive specific properties which are only useful in alter table?